jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/XmlInheritanceProcessor.java

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright 2019 Haulmont.
```

Comment:
Flow UI copyright a slightly different. Please copy copyright from one of Flow UI files.